### DISCUSSION TITLE
Ask for input during a running workflow

### DISCUSSION THREAD

**Discussion by Howe829** [langgenius/dify#8442] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

I am wondering how to ask for user input during a running workflow. For example, if the first LLM node generates multiple choices, I want to let the user choose which option to use for the second LLM input.

**Comment by Ima-Beast** [comment]
As far as I know, a workflow is a form of stream processing and currently does not support interacting with the user during its execution. You could consider breaking it into two applications, with user interaction added between them. Alternatively, you might also consider adding a parameter during the initial user input process. Based on this parameter and the first output from the large model, you can use code to determine the second input to the large model. Hope this helps.

**Comment by FB886** [comment]
I tried to use a workflow to complete a complex conversation that involves loops and conditionals. However, the current workflow only supports one-way flow from the start node, and there is no way to receive user input again in the conversation loop. I hope to add a link node before the start node, so that I can control the entire process.

**Reply by amroeser** [reply]
is there an update about this feature?
